frontend/src/audios.html

```diff
@@ -20,8 +29,13 @@
       <div id="year-list"></div>
       <div id="location-list"></div>
     </div>
-    <div class="flex-none border-t p-2 md:hidden">
-      <button data-filter-toggle class="w-full rounded-md bg-neutral-600 p-2 font-semibold uppercase text-white shadow-sm">Close</button>
+    <div class="flex-none border-t p-2 align-top md:hidden">
```

Comment:
Why would we need to add `align-top` here? It was working without it before.

Reply:
align-top is a typo, removed